specification/columns/billingperiodend.md

```diff
@@ -2,7 +2,11 @@
 
 Billing Period End represents the end date and time of the [*billing period*](#glossary:billing-period).
 
-The BillingPeriodEnd column MUST be present in the billing data. This column MUST be of type Date/Time and MUST NOT contain null values. BillingPeriodEnd column MUST conform to [Date/Time Format](#date/timeformat) requirements. The sum of the [BilledCost](#billedcost) column for [*rows*](#glossary:row) in a given *billing period* MUST match the sum of the invoices received for that *billing period* for a [*billing account*](#glossary:billing-account).
+The BillingPeriodEnd column MUST be present in the billing data. This column MUST be of type Date/Time and MUST NOT contain null values.
+
+BillingPeriodEnd column MUST conform to an [Exclusive](#glossary:exclusivebound) [Date/Time Format](#date/timeformat) requirements. The sum of the [BilledCost](#billedcost) column for [*rows*](#glossary:row) in a given *billing period* MUST match the sum of the invoices received for that *billing period* for a [*billing account*](#glossary:billing-account).
+
+For example, a time period where BillingPeriodStart is '2024-01-01T00:00:00Z' and BillingPeriodEnd is '2024-02-01T00:00:00Z' only includes charges for January, given BillingPeriodEnd is *excluded* from the period.
```

Comment:
Should the example be a part of the description paragraph so we can leave the normative content as one paragraph at the end? Just styling-wise, this seems a bit different from our standard approach